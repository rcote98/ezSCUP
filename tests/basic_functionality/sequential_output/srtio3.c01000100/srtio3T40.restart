2	2	2
5	3
Sr	Ti	O
0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	1	1	1.58056792E-03	1.39117402E-02	-1.39031150E-02
0	0	0	2	2	1.38814963E-02	-1.91755517E-02	-5.33018612E-03
0	0	0	3	3	2.79131687E-01	2.64579281E-01	1.46747890E-02
0	0	0	4	3	-3.93926981E-02	-4.31209033E-03	-2.86675684E-01
0	0	0	5	3	-5.33999511E-04	6.27256639E-02	-2.97660474E-01
0	0	1	1	1	-2.15732233E-02	-4.20347421E-03	-6.39306187E-03
0	0	1	2	2	-4.21498568E-03	1.59698844E-02	-2.12093794E-03
0	0	1	3	3	-2.78909354E-01	-3.15268113E-01	1.08041284E-02
0	0	1	4	3	6.61344111E-03	-4.44726258E-03	3.12840181E-01
0	0	1	5	3	2.95171876E-03	-1.87566671E-03	2.89828651E-01
0	1	0	1	1	-1.87849539E-03	8.84742371E-03	1.73540336E-02
0	1	0	2	2	-1.09306074E-02	-5.95359228E-03	5.13151931E-03
0	1	0	3	3	-2.73471358E-01	-3.06342390E-01	2.65872158E-03
0	1	0	4	3	6.44186299E-02	-3.22146420E-03	3.36403460E-01
0	1	0	5	3	2.26755589E-03	-4.16425041E-02	3.17236780E-01
0	1	1	1	1	7.82224533E-03	2.18963969E-03	-5.41015384E-03
0	1	1	2	2	-8.67204191E-03	1.17126460E-02	-4.23422868E-03
0	1	1	3	3	2.88955149E-01	2.93139896E-01	1.04826469E-02
0	1	1	4	3	4.00196051E-03	-1.20788587E-02	-2.82183778E-01
0	1	1	5	3	9.14864144E-03	6.12771252E-03	-3.05672158E-01
1	0	0	1	1	8.24263425E-03	1.05435730E-03	-1.92442718E-02
1	0	0	2	2	2.95186123E-03	9.42202179E-03	-1.58500760E-02
1	0	0	3	3	-2.78711119E-01	-3.02622770E-01	-1.70367883E-02
1	0	0	4	3	3.37601666E-02	-7.46966899E-03	3.20397280E-01
1	0	0	5	3	5.51227519E-03	-5.91722714E-02	2.85968627E-01
1	0	1	1	1	6.37378224E-03	4.60137619E-03	3.38558800E-03
1	0	1	2	2	4.54961339E-03	7.18384927E-03	-1.00832812E-02
1	0	1	3	3	2.91139937E-01	3.10088403E-01	-6.32932689E-03
1	0	1	4	3	1.69293067E-03	-2.92576500E-03	-3.31434162E-01
1	0	1	5	3	3.12418270E-03	9.56065958E-03	-3.13216888E-01
1	1	0	1	1	-5.80570611E-03	9.69154228E-03	-1.10712979E-03
1	1	0	2	2	-1.76532587E-02	7.33817389E-03	-3.71422524E-03
1	1	0	3	3	2.78558082E-01	2.95666817E-01	-6.17282944E-03
1	1	0	4	3	-6.88059119E-02	9.41743950E-03	-3.09521658E-01
1	1	0	5	3	-1.66735414E-03	4.92671535E-02	-2.91524160E-01
1	1	1	1	1	-1.54208581E-02	1.35000781E-02	-2.70989678E-03
1	1	1	2	2	-1.39714055E-02	1.22191219E-03	-4.05448508E-03
1	1	1	3	3	-2.88921908E-01	-3.17302977E-01	-4.80354933E-04
1	1	1	4	3	1.71266772E-02	-9.13355535E-03	3.13647701E-01
1	1	1	5	3	-3.27094921E-03	9.93030479E-03	3.01249206E-01
