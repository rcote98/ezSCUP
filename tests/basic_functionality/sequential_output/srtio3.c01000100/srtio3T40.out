
*****************************************************************
*                          Welcome to                           *
*                           SCALE-UP                            *
*                                                               *
*            Second-Principles Computational Approach           *
*                             for                               *
*                     Lattice and Electrons                     *
*                                                               *
*                       Version:         1.0.0.Gorbea           *
*                    Build date: 20200801T181                   *
*****************************************************************
*                    SCALE-UP is property of                    *
*              the University of Cantabria (Spain)              *
*    and the Luxembourg Institute of Science and Technology     *
*****************************************************************
* Authors:                                                      *
*                                                               *
*  - Pablo Garcia-Fernandez   garciapa@unican.es                *
*  - Jorge Iñiguez            jorge.iniguez@list.lu             *
*  - Javier Junquera          junqueraj@unican.es               *
*****************************************************************
* SCALE-UP is based on two earlier programs:                    *
*                                                               *
* -  SPLD ('Second-Principles Lattice Dynamics'), developed     *
*    at the ICMAB-CSIC (Spain) by J.C. Wojdel, M.P. Ljungberg   *
*    and J. Iñiguez, and distributed under MIT license.         *
*                                                               *
* -  MKII a second-principles electron code                     *
*    developed at the University of Cantabria by                *
*    P. Garcia-Fernandez  and J. Junquera.                      *
*****************************************************************
*                                                               *
* Bibliography:                                                 *
*                                                               *
*  P. Garcia-Fernandez et al. Phys. Rev. B 93, 195137 (2016)    *
*  J. C. Wojdel et al., J. Phys. Cond. Matt. 25, 305401 (2013)  *
*****************************************************************
Reading parameter file... lattice... done!
Reading restart file     srtio3T40.restart                                                               

Basic data:
  Number of CPUs:               1
  Run mode:            Montecarlo
  Lattice:                 Active
  Electrons:           NOT Active
  External electric field, static. Field components (V/m):   0.000000E+00   0.000000E+00   0.000000E+00

Reference system:
 Supercell data:
  Lattice constants (Ang):   7.690   7.690   7.690
  Lattice angles (degree):  90.000  90.000  90.000
 Supercell size:   2   2   2

 Total number of atoms          :      40
 Total number of orbitals       :       0
 Total number of active orbitals:       0

 Atom in unit cell:
 Atom   Label       Mass  Charge 
    1   Sr         87.60     0.0
    2   Ti         47.90     0.0
    3   O          16.00     0.0
    4   O          16.00     0.0
    5   O          16.00     0.0

 Orbitals in unit cell:
 Orbital   Label  

 Background dielectric tensor:
    6.2319     0.0000     0.0000
    0.0000     6.2319    -0.0000
    0.0000    -0.0000     6.2319

Initial values (basic output MC simulation)
Energy (eV):            -0.298487E+05
Polarization (C/m2):    -0.008899    0.011425   -0.010006
Strain:                  0.000000    0.000000    0.000000    0.000000    0.000000    0.000000

LT:    Iter       Temp(K)    Energy(eV)   Pol_x(C/m2)   Pol_y(C/m2)   Pol_z(C/m2) Av_Pl_x(C/m2) Av_Pl_y(C/m2) Av_Pl_z(C/m2)       Strn_xx       Strn_yy       Strn_zz       Strn_yz       Strn_xz       Strn_xy    Av_Strn_xx    Av_Strn_yy    Av_Strn_zz    Av_Strn_yz    Av_Strn_xz    Av_Strn_xy 
LT:      10  0.400000E+02 -0.298485E+05  0.362587E-01  0.704222E-02 -0.146645E-02  0.105177E-01  0.228058E-01  0.395681E-02  0.000000E+00  0.000000E+00 -0.745788E-03 -0.288116E-02  0.427649E-02  0.000000E+00  0.000000E+00  0.000000E+00  0.400320E-03 -0.108729E-02  0.235336E-02  0.000000E+00 
LT:      20  0.400000E+02 -0.298485E+05  0.573406E-02 -0.245113E-01 -0.187210E-01  0.178452E-01  0.427248E-02 -0.575654E-02  0.000000E+00  0.000000E+00 -0.240305E-02 -0.188084E-02  0.441619E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.441024E-03 -0.142669E-02  0.352808E-02  0.000000E+00 
LT:      30  0.400000E+02 -0.298486E+05 -0.109160E-01 -0.138323E-01 -0.168942E-02  0.118328E-01 -0.357927E-02 -0.411930E-02  0.000000E+00  0.000000E+00 -0.272661E-02 -0.392561E-02  0.186317E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.104823E-02 -0.173810E-02  0.342381E-02  0.000000E+00 
LT:      40  0.400000E+02 -0.298486E+05  0.100931E-01  0.243581E-01  0.341761E-01  0.787030E-02 -0.252646E-02  0.410236E-02  0.000000E+00  0.000000E+00 -0.359425E-02 -0.472477E-02  0.380983E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.156471E-02 -0.240285E-02  0.339503E-02  0.000000E+00 
LT:      50  0.400000E+02 -0.298486E+05  0.372734E-01  0.414492E-01 -0.138703E-01  0.889629E-02  0.475010E-02  0.453731E-02  0.000000E+00  0.000000E+00 -0.357865E-02 -0.491891E-02  0.398428E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.208587E-02 -0.304805E-02  0.327913E-02  0.000000E+00 
LT:      60  0.400000E+02 -0.298485E+05 -0.523595E-01  0.158587E-01 -0.217984E-01  0.577100E-02  0.903591E-02  0.436503E-02  0.000000E+00  0.000000E+00  0.657141E-04 -0.506770E-02 -0.492598E-03  0.000000E+00  0.000000E+00  0.000000E+00 -0.181161E-02 -0.316017E-02  0.310806E-02  0.000000E+00 
LT:      70  0.400000E+02 -0.298486E+05 -0.719396E-02 -0.570800E-02  0.222758E-02  0.283917E-02  0.902124E-02  0.276551E-02  0.000000E+00  0.000000E+00  0.324956E-03 -0.132912E-02  0.103066E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.141903E-02 -0.292535E-02  0.255742E-02  0.000000E+00 
LT:      80  0.400000E+02 -0.298485E+05  0.720809E-02 -0.159750E-01 -0.166056E-01  0.336782E-02  0.684278E-02  0.174891E-02  0.000000E+00  0.000000E+00 -0.223167E-02 -0.590817E-02  0.216402E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.144620E-02 -0.320114E-02  0.242277E-02  0.000000E+00 
LT:      90  0.400000E+02 -0.298486E+05 -0.688232E-02 -0.187352E-01 -0.200305E-01  0.326224E-02  0.450810E-02  0.530087E-04  0.000000E+00  0.000000E+00 -0.455162E-02 -0.495298E-02 -0.338619E-03  0.000000E+00  0.000000E+00  0.000000E+00 -0.171822E-02 -0.320363E-02  0.228591E-02  0.000000E+00 
LT:     100  0.400000E+02 -0.298486E+05  0.122612E-01 -0.402568E-01  0.154096E-01  0.395651E-02  0.306267E-02  0.108983E-03  0.000000E+00  0.000000E+00 -0.492729E-02 -0.284273E-02  0.187851E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.213906E-02 -0.338307E-02  0.214863E-02  0.000000E+00 

Final values (basic output MC simulation)
Energy (eV):            -0.298486E+05
Polarization (C/m2):     0.012261   -0.040257    0.015410
Strain:                  0.000000    0.000000   -0.004927   -0.002843    0.001879    0.000000

-----------------------------------------------------------------
 Backing up the input read by fdf:

System_name     srtio3T40
Print_routineinfo         5     # default value
Print_routinedepth         0     # default value
Print_inout         0     # default value
Print_interactions         0     # default value
Print_k-vectors         0     # default value
Print_energies         1     # default value
Print_wavefunctions        -1     # default value
Print_electrostatic         1     # default value
Print_orbocc         1     # default value
Print_bands        -1     # default value
Print_forces        -1     # default value
Print_detail        -1     # default value
Print_Just_Geometry        -1     # default value
Print_Velocity         0     # default value
Print_Space_Hamiltonian        -1     # default value
Print_space_denmat        -1     # default value
Print_Geometry         1     # default value
Print_geoformat     normal     # default value
Print_test_file        -1     # default value
Print_local_lattice_energy        -1     # default value
Print_local_current        -1     # default value
Print_std_electron_nsteps         1     # default value
Print_std_lattice_nsteps             10
Print_std_energy              T
Print_std_polarization              T
Print_std_current              F     # default value
Print_std_strain              T
Print_std_av_polarization              T
Print_std_av_strain              T
Print_std_av_energy              F     # default value
Print_std_delta_energy              F     # default value
Print_std_temperature              T     # default value
Print_std_maximum_force              F     # default value
Print_std_maximum_stress              F     # default value
Print_std_mdd              T     # default value
Print_std_fermi              T     # default value
Print_std_pulay              T     # default value
Print_std_neh              T     # default value
Print_std_el_pol              T     # default value
Print_std_micro              F     # default value
run_mode     monte_carlo
Print_std_mdd              F     # default value
Print_std_fermi              F     # default value
Print_std_pulay              F     # default value
Print_std_neh              F     # default value
Print_std_electron_nsteps     10000     # default value
No_electron
%block fix_strain_component
 T T F F F T
T                                                                                             T
T                                                                                             T
F                                                                                             F
F                                                                                             F
F                                                                                             F
T                                                                                             T
Parameter_file     srtio3_full_lat.xml
%block Supercell
 2 2 2
%endblock Supercell
electric_field_scale          1.000000000     v/m     # default value
%block static_electric_field
 0.0 0.0 0.0
temperature         0.1000000000E-98 kelvin     # default value
%block external_stress
 0.0 0.0 0.0 0.0 0.0 0.0
Charge_tolerance         0.1000000000E-07     # default value
Geometry_restart
Geometry_restart     srtio3T40.restart
MC_nsweeps
MC_nsweeps            100
MC_temperature
MC_temperature          40.00000000         kelvin
# above item originally: MC_temperature          40.00000000     kelvin
MC_annealing_rate          1.000000000
MC_init_step_d         0.9448629925E-01 bohr     # default value
MC_init_step_s         0.1889725985E-02 bohr     # default value
MC_max_step_d         0.9448634389           bohr
# above item originally: MC_max_step_d         0.5000000000     ang
MC_min_step_d         0.1889725985E-05 bohr     # default value
MC_target_acc_d         0.5000000000         # default value
MC_target_acc_s         0.5000000000         # default value
MC_strains              T
n_write_mc
n_write_mc             10
Print_std_lattice_nsteps
*****************************************************************
 Memory usage:
  Function             Total_memory(Mb)  Max_memory(Mb)      Node
  setup_main                     0.0256          0.0256         0
  setup_basicvariables           0.0056          0.0056         0
  setup_latt_hamil               0.0014          0.0014         0
  setup_init_data_node           0.0256          0.0256         0
  hel_setup_structfactors        0.0256          0.0256         0
  hlt_create_supercell           0.0014          0.0014         0

 Timing statistics (seconds)
  Function                       Wall   Max_Wall    Max_CPU  Node
  setup_main                     3.81       3.81       3.76     0
  setup_readinput                3.58       3.58       3.54     0
  montecarlo_main                0.21       0.21       0.21     0
  montecarlo_sweep               0.20       0.20       0.20     0
  io_readparameterfile           3.58       3.58       3.54     0
  setup_latt_hamil               0.23       0.23       0.22     0
  hlt_add_harmonic_didi          0.22       0.22       0.22     0

 Total memory usage:     0.03 MB
 Total WALL:           4.022 seconds
*****************************************************************
*                                                               *
*               Succesfully ending SCALE-UP                     *
*                                                               *
*****************************************************************
